### ISSUE TITLE
CoronaCoin Token 

### ISSUE THREAD

**Issue by howlchan** [Uniswap/uniswap-frontend#659]
**Please provide the following information for your token.**

Token Address: 0x10ef64cb79fd4d75d4aa7e8502d95c42124e434b
Token Name (from contract): CoronaCoin
Token Decimals (from contract): 18
Token Symbol (from contract): NCOV
Uniswap Exchange Address of Token: 0x10ef64cb79fd4d75d4aa7e8502d95c42124e434b

Link to the official homepage of token: https://coronatoken.org
Link to CoinMarketCap or CoinGecko page of token: https://www.coingecko.com/en/coins/coronacoin

Some tokens (e.g. BNB) do not work with Uniswap v1. In order to assess if your token works correctly, please complete small-value transactions of each of the types below, and submit the Etherscan transaction links for our review.
Test `addLiquidity` transaction: 
Test `swap` transaction: 
Test `removeLiquidity` transaction: 

Are you willing to add liquidity to the liquidity pool for this token? (Y/N): Y
If so, how much liquidity are you willing to add?: 2 ETH

**Comment by DiegoJRR** [comment]
Finally

**Comment by Tech1k** [comment]
I second adding Coronacoin, it's  great coin with a great purpose!

**Comment by CouldntDeal** [comment]
I agree.

**Comment by victorjvs** [comment]
I agree with you! CoronaCoin is having exceptional volume and has a great cause behind it!

**Comment by Abubakrdammy** [comment]
Hopefully, this project will be permanently listed on Uniswap.
I am very sure it will also contribute to the amount of Uniswap's users.

**Comment by boredomware** [comment]
Great idea
